build_projects/shared-build-targets-utils/Utils/BuildVersion.cs

```diff
@@ -7,7 +7,7 @@ public class BuildVersion : Version
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
-        public string NetCoreAppVersion => $"{Major}.{Minor}.{Patch}-rc3-{CommitCountString}";
+        public string NetCoreAppVersion => $"{Major}.{Minor}.{Patch}-{ReleaseSuffix}-{CommitCountString}";
         public string ProductionVersion => $"{Major}.{Minor}.{Patch}";
```

Comment:
This problem is actually fixed as part of my larger https://github.com/dotnet/core-setup/pull/14, but that is not yet ready to merge. I have to figure out how to fix the other versions but I can work on getting at least this aspect of it fixed.